frontend/packages/kubevirt-plugin/src/selectors/vm/selectors.ts

```diff
@@ -27,11 +33,11 @@ export const getInterfaces = (vm: VMKind, defaultValue = []): V1NetworkInterface
     ? defaultValue
     : vm.spec.template.spec.domain.devices.interfaces;
 
-export const getNetworks = (vm: VMKind, defaultValue = []): V1Network[] =>
+export const getNetworks = (vm: VMLikeEntityKind, defaultValue = []): V1Network[] =>
```

Comment:
This will not work for `Template`, structure differs.
Same comment for all similar changes bellow.